about/maintainer_policy/index.markdown

```diff
@@ -19,6 +19,7 @@ In addition to gratitude, some advantages of being a MoveIt maintainer / core co
  - Be listed on the MoveIt website
  - Networking opportunities: gain reputation in the largest robotics community on earth
  - Quicker code contributions: you only need one review to get your code merged in, while non-maintainer/non-core contributor pull requests require two approvals.
+ - Free T-Shirt from PickNik Robotics
```

Comment:
Is it a "PickNik" T-Shirt or do we give them a "MoveIt"-Shirt?

Reply:
MoveIt shirt.
```suggestion
 - Free MoveIt T-Shirt from PickNik Robotics
```